packages/template-typescript-minimal/config/default.js

```diff
@@ -0,0 +1,234 @@
+                                    },
+                                    {
+                                        id: 'fr-FR',
+                                        preferredCurrency: 'EUR'
+                                    },
+                                    {
+                                        id: 'it-IT',
+                                        preferredCurrency: 'EUR'
+                                    },
+                                    {
+                                        id: 'ja-JP',
+                                        preferredCurrency: 'JPY'
+                                    },
+                                    {
+                                        id: 'ko-KR',
+                                        preferredCurrency: 'KRW'
+                                    },
+                                    {
+                                        id: 'pt-BR',
+                                        preferredCurrency: 'BRL'
+                                    },
+                                    {
+                                        id: 'zh-CN',
+                                        preferredCurrency: 'CNY'
+                                    },
+                                    {
+                                        id: 'zh-TW',
+                                        preferredCurrency: 'TWD'
+                                    }
+                                ],
+                                defaultLocale: 'en-GB'
+                            }
+                        }
+                    ],
+                    staleWhileRevalidate: 900,
+                    url: {
+                        site: 'path',
+                        locale: 'path',
+                        showDefaults: true,
+                        interpretPlusSignAsSpace: false
+                    }
+                }
+            ],
+            [
+                '@salesforce/extension-commerce-bm-seo',
+                {
+                    enabled: true,
+                    path: '/sample-page',
+                    commerceAPI: {
+                        proxyPath: '/mobify/proxy/api',
+                        parameters: {
+                            clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
+                            organizationId: 'f_ecom_zzrf_001',
+                            shortCode: '8o7m175y',
+                            siteId: 'RefArchGlobal'
+                        }
+                    },
+                    commerceAPIAuth: {
+                        propertyNameInLocals: 'commerceAPIAuth'
+                    },
+                    resourceTypeToComponentMap: {
+                        category: 'ProductList',
+                        product: 'ProductDetail',
+                        // TODO: what component should content be mapped to?
+                        content_asset: 'ProductList'
+                    }
```

Comment:
@jeremy-jung1 here is where the config for the urlMapping resourceType is mapped to the component name. We could have this added to the default.json